scripts/build_lbann_lc.sh

```diff
@@ -807,6 +787,9 @@ CONFIGURE_COMMAND=$(cat << EOF
 -D LBANN_SB_BUILD_ALUMINUM=${WITH_ALUMINUM} \
 -D ALUMINUM_ENABLE_MPI_CUDA=${ALUMINUM_WITH_MPI_CUDA} \
 -D ALUMINUM_ENABLE_NCCL=${ALUMINUM_WITH_NCCL} \
+-D LBANN_SB_BUILD_CONDUIT=${WITH_CONDUIT} \
+-D LBANN_SB_BUILD_HDF5=${WITH_CONDUIT} \
```

Comment:
We might use HDF5 in some other context at the same time. e.g., check-pointing #462 , which is unrelated to the conduit use.

Reply:
If none of this functionality exists at the head of develop when this gets approved/merged, I would argue that such a need does not exist at this time and the appropriate changes can be incorporated into the build system as part of those pending or future PRs.